src/global/helpers/eventing/input-modality-detector.ts

```diff
@@ -76,7 +76,7 @@ class SbbInputModalityDetector {
   public get mostRecentModality(): SbbInputModality | null {
     return this._mostRecentModality;
   }
-  private _mostRecentModality: SbbInputModality | null = null;
+  private _mostRecentModality: SbbInputModality | null = 'mouse';
```

Comment:
should we add a test case somehow and make a comment here why we use mouse as a default?